src/custom/constants/chainInfo/chainInfoMod.ts

```diff
@@ -56,7 +55,7 @@ export type ChainInfoMap = { readonly [chainId: number]: L1ChainInfo | L2ChainIn
 } & { readonly [chainId in SupportedL1ChainId]: L1ChainInfo }
 
 export const CHAIN_INFO: ChainInfoMap = {
-  ...UNI_CHAIN_INFO,
+  // ...UNI_CHAIN_INFO,
```

Comment:
Why comment this out?

Reply:
Because we don't need it. We have all values for all supported chains here